wordpress-mobile/WordPress-Android
# Wpcom-only subdomain suggestions in new site-creation

Fixes #7204 

This PR will be updated with the specific FluxC hash instead of the branch snapshot dependency after https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/707 gets merged.

To test:
1. With the app logged in wpcom, create a new wpcom site via the site picker
2. Reach the domain selection screen
3. Notice that the already given suggestions are only for wpcom subdomains
4. Enter a query text to trigger a new query
5. Notice that the given suggestions are only for wpcom subdomains
